source/administration/config-settings.rst

```diff
@@ -1601,9 +1601,9 @@ Note: OAuth 2.0 applications can be authorized by all users if they have the **C
 
 Enable integrations to override usernames
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-**True**: Webhooks, slash commands and other integrations, such as `Zapier <https://docs.mattermost.com/integrations/zapier.html>`_, will be allowed to change the username they are posting as.
+**True**: Webhooks, slash commands, and other integrations, such as `Zapier <https://docs.mattermost.com/integrations/zapier.html>`_, will be allowed to change the username they are posting as. If no username is present, the username for the post is the same as it would be for a setting of **False**.
 
-**False**: Webhooks, slash commands and OAuth 2.0 apps can only post as the username of the account they were set up with. See http://mattermost.org/webhooks for more details.
+**False**: Custom slash commands and OAuth 2.0 apps can only post as the username of the user who used the slash command or OAuth 2.0 app. For incoming webhooks and outgoing webhooks, the username is "webhook". See http://mattermost.org/webhooks for more details.
```

Comment:
@JeffSchering I did some testing, and posts made by a Zapier OAuth2 integration post as the user who set up the integration. 

Also confirmed that custom slash commands post with the username of the user who posted the slash command in the channel (not necessarily the user who created the slash command). Confirmed that incoming webhook posts as the user who set up the webhook.

Do you want me to test further? Or is that enough info to verify?